publish.sh

```diff
@@ -13,3 +13,8 @@ git checkout -b $publish_branch
 git push origin $publish_branch
 
 npx lerna publish
+
+# Update package-lock.json files as well
+npx lerna exec npm install
+git commit --all --amend --no-edit
```

Comment:
Does lerna not tie tags to a particular commit? Would this not destroy the tags?